shims/spark31/src/main/scala/com/nvidia/spark/rapids/shims/spark31/Spark31ShimServiceProvider.scala

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2020, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.nvidia.spark.rapids.shims.spark31
+
+import com.nvidia.spark.rapids.{SparkShims, SparkShimServiceProvider}
+
+class Spark31ShimServiceProvider extends SparkShimServiceProvider {
+
+  val SPARK31VERSIONNAME = "3.1.0-SNAPSHOT"
```

Comment:
Worth updating this now so it can work with either the snapshot or the 3.1.0 release version?  Otherwise we'll have to remember to update this when 3.1.0 ships.